protoc.version

```diff
@@ -0,0 +1 @@
+libprotoc 3.0.0
```

Comment:
Can you update to 3.5.0?

Reply:
um, I guess. I wasn't really looking forward to updating my protoc for a PR 😜 .